### ISSUE TITLE
Refactoring: Rename temporary could skip the confirmation

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#13097]
Rename temporary refactoring is really simple and you know that the scope will never be outside the current method in which we are. 

I always have the feeling that the confirmation dialog for this refactoring never help me and always slow me down. 

Maybe just for this refactoring we could skip it?

**Comment by privat** [comment]
Yes, but this refactoring is broken anyway because it does not work on the currently edited method, and wants to run on the old compiled version. If you proceed after the useless preview, you might lose your editions.

**Comment by jecisc** [comment]
I've noticed that. 
But I'm still using it a lot ahah And sometimes getting bitten by what you said :C

**Comment by jordanmontt** [comment]
Yes, it refactors the last compiled method. I do not if this is a bug of Calypso or the refactors.

As for this issue, I agree! In other IDEs renaming temps don't ask for a confirmation :)

**Comment by jordanmontt** [comment]
We were checking with @ValentinBourcier this issue. We found out that class SycRenameArgOrTempCommand is the one that calls this refactor. After, one of the superclass CmdCommandhas a method confirmRefactoringInContext: by: calls the refactoring preview presenter (see image). And then, that presenter in the method accept applies the refactor. So, this is very hard coded.

We think that the best is to move this logic to the driver @balsa-sarenac
The refactors should be able to call the driver with the option "no confirmation" and the driver is the one that should be responsible of applying the refactor with or without presenter. We also think that the logic of applying the refactors should be moved from SycRenameArgOrTempCommand>>accept to the driver because now the presenter it also has business logic.

<img width="498" alt="Capture d’écran 2023-03-31 à 11 38 21" src="https://user-images.githubusercontent.com/33934979/229084799-79b84a0e-1181-4054-954a-45a19e634542.png">

**Comment by jecisc** [comment]
This is done and we can get it back with a setting!
